client/components/shipping/packages/add-package.js

```diff
@@ -79,7 +87,6 @@ const useDefaultField = ( value, updatePackagesField ) => {
 		is_letter: 'envelope' === value,
 		name: null,
 		is_user_defined: true,
-		outer_dimensions: null,
```

Comment:
In the case of no outer_dimensions, we want to delete that member altogether. If we save `packageData` with `outer_dimensions` equal to null or empty string, the form validation will fail.